lerna.json

```diff
@@ -1,4 +1,5 @@
 {
+  "$schema": "./lerna.schema.json",
```

Comment:
How do others outside of this repo consume the schema?

Reply:
Via relative path or URL: "./node_modules/lerna/leran.schema.json" or "unpkg.com/..."